db/schema.rb

```diff
@@ -87,6 +87,7 @@
     t.text 'partner_info'
     t.string 'deadline_to_consign'
     t.string 'sale_location'
+    t.integer 'offer_responses_count'
```

Comment:
[non-blocking] - for my education - this is associated with adding `counter_cache` to the model, right?